bmeg_app/__main__.py

```diff
@@ -0,0 +1,171 @@
+from bmeg_app.app import app
```

Comment:
missing-module-docstring http://pylint-messages.wikidot.com/messages:c0111

Reply:
Comments added in commit 
90e4d56